### ISSUE TITLE
Windows Ganache GUI not taking updates from Truffle running on WSL2

### ISSUE THREAD

**Issue by Suedo** [trufflesuite/ganache#2141]
I have added truffle workspace to Ganache as usual. But now when I do a simple `truffle develop` followed by `migrate` inside the truffle shell, Ganache says that the contracts are deployed, but it doesn't pickup changes to the wallet balances. 

My `truffle-config.js` files has only one change:
```js
  compilers: {
    solc: {
      version: "native", 
    },
  },
```
I am running truffle inside WSL2, whereas ganache gui is installed natively in windows.

Below is the balance reported by Truffle shell:

```cmd
truffle(develop)> web3.eth.getBalance(accounts[0])
'99934183260000000000'

truffle(develop)> accounts[0]
'0x6b318B9350E42bEB94f9Ae5e8b9DA854a6137628'
```
Where that same account `0x6b318B9350E42bEB94f9Ae5e8b9DA854a6137628` in Ganache shows balance full 100 Eth:
![image](https://user-images.githubusercontent.com/4963779/99699566-18010080-2ab8-11eb-808e-6e620eaf6208.png)

**Comment by davidmurdoch** [comment]
You might need to add

```
networks: {
   development: {
     host: "127.0.0.1",
     port: 9545,
     network_id: "*"
   }
}
```

to your truffle config.

**Comment by Suedo** [comment]
> You might need to add
> 
> ```
> networks: {
>    development: {
>      host: "127.0.0.1",
>      port: 9545,
>      network_id: "*"
>    }
> }
> ```
> 
> to your truffle config.

I Have tried this, but it does not work

**Comment by davidmurdoch** [comment]
Are you running `truffle console --network development` to connect to the running Ganache UI instance?

**Comment by Suedo** [comment]
Yes, I am doing exactly the same. The usual steps. It works when I have both truffle and ganache working natively in windows, but when truffle is inside wsl2, ganache cannot connect to it

**Comment by davidmurdoch** [comment]
This sounds like it is a Truffle issue. I'll transfer this issue to the truffle repo.
